dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/DotNet/Internal/Http/Specs/CreateCollectionRequest.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
```

Comment:
Suggestion: rename namespace to HttpSchema to match the conventions in this repository

Reply:
Updated.